File: app/src/main/java/hu/epam/worktime/mvvmworkdroid/di/qualifiers/ApplicationContext.java
package hu.epam.worktime.mvvmworkdroid.di.qualifiers;

import javax.inject.Qualifier;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * Created by Mihaly_Hunyady on 2016. 11. 24..
 */

@Qualifier
@Retention(RetentionPolicy.RUNTIME)
public @interface ApplicationContext {
}

File: app/src/main/java/hu/epam/worktime/mvvmworkdroid/di/qualifiers/ApplicationContext.kt
package hu.epam.worktime.mvvmworkdroid.di.qualifiers

import java.lang.annotation.RetentionPolicy
import javax.inject.Qualifier
import kotlin.annotation.Retention

/**
 * Created by Mihaly_Hunyady on 2016. 11. 24..
 */

@Qualifier
@Retention(AnnotationRetention.RUNTIME)
annotation class ApplicationContext
